cli.js

```diff
@@ -79,9 +79,14 @@ async function init() {
 	}
 
 	const appName = appInfo.CFBundleDisplayName || appInfo.CFBundleName;
-	const dmgTitle = appName.length > 27 ? (cli.flags.dmgTitle || appName) : appName;
+	const dmgTitle = cli.flags.dmgTitle || appName;
 	const dmgPath = path.join(destinationPath, `${appName} ${appInfo.CFBundleShortVersionString}.dmg`);
 
+	if (cli.flags.dmgTitle > 27) {
+		console.error('Disk image title exceeds 27 characters, aborting');
```

Comment:
```suggestion
		ora.fail('Disk image title exceeds 27 characters, aborting');
```

And I think the text should explain why more than 27 characters are not allowed.